jrpc/options_test.go

```diff
@@ -22,4 +35,13 @@ func TestOptions(t *testing.T) {
 
 	assert.Len(t, def.CustomHeadersKey, 2)
 	assert.Equal(t, def.ReadHeaderTimeout.String(), dur.String())
+	require.NotNil(t, def.CorsOptions)
+	assert.Len(t, def.CorsOptions.AllowedOrigins, 1)
+	assert.Equal(t, "*", def.CorsOptions.AllowedOrigins[0])
```

Comment:
This line has already explicitly passed the one above, so checking the length is redundant. To simplify the tests, checking the whole value is sufficient.